ArrayIndexOutOfBoundsException in harvestcraft.zs

#10 broke harvestcraft.zs

https://github.com/sebinside/SkyblockNachfolger/blame/master/scripts/harvestcraft.zs#L35

```log
[INITIALIZATION][CLIENT][ERROR] [crafttweaker]: Error executing {[0:crafttweaker]: harvestcraft.zs}: 7
java.lang.ArrayIndexOutOfBoundsException: 7
	at Harvestcraft.__script__(harvestcraft.zs:35)
	at __ZenMain__.run(Harvestcraft)
	at crafttweaker.runtime.CrTTweaker.loadScript(CrTTweaker.java:228)
	at crafttweaker.runtime.CrTTweaker.loadScript(CrTTweaker.java:105)
	at crafttweaker.mc1120.events.CommonEventHandler.registerRecipes(CommonEventHandler.java:73)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_274_CommonEventHandler_registerRecipes_Register.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:857)
	at net.minecraftforge.common.crafting.CraftingHelper.loadRecipes(CraftingHelper.java:636)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:747)
	at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:336)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:535)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

--- a/scripts/harvestcraft.zs
+++ b/scripts/harvestcraft.zs
@@ -32,5 +32,5 @@ mods.botania.Apothecary.addRecipe(<harvestcraft:shadedgarden>, [<ore:petalLime>,
 
 val harvestCraftCycle = [<harvestcraft:shadedgarden>, <harvestcraft:windygarden>, <harvestcraft:soggygarden>, <harvestcraft:tropicalgarden>, <harvestcraft:aridgarden>, <harvestcraft:frostgarden>, <harvestcraft:shadedgarden>] as IItemStack[];
 for i, item in harvestCraftCycle {
-	mods.botania.ManaInfusion.addAlchemy(harvestCraftCycle[i--1], item, 50);
+	mods.botania.ManaInfusion.addAlchemy(item, harvestCraftCycle[(i+1) % harvestCraftCycle.length], 50);
 }

